boards/riscv/hpm6750evkmini/support/probes/ft2232.cfg

```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 hpmicro
+# SPDX-License-Identifier: BSD-3-Clause
+
+bindto 0.0.0.0
+adapter speed 10000
+reset_config trst_and_srst
+adapter srst delay 50
+
+adapter driver ftdi
+ftdi_vid_pid 0x0403 0x6010
+
+ftdi_layout_init 0x0208 0x020b
+ftdi_layout_signal nTRST -data 0x0200 -noe 0x0400
+ftdi_layout_signal nSRST -data 0x0100 -noe 0x0800
+
```

Comment:
This file (and other cfgs as well) seems to be generic. Is it not provided by openocd by default?

Reply:
These configurations for io are inconsistent from chip to chip.